update rubies list in appveyor build matrix

miltador opened the pull request:
Added available ruby versions to the build matrix of AppVeyor builds to ensure things are going fine on Windows. I saw the comment about nokogiri issues but I wonder how things are going on now.

miltador commented:
Ahh, it's disabled on this repository. Are there any issues with AppVeyor build config? I would like to help with it if needed.

miltador commented:
Enabled AppVeyor for my fork to test out configuration, here are the results: https://ci.appveyor.com/project/miltador/asciidoctor

mojavelinux commented:
We hadn't done this before since we didn't see the need. I suppose these nuances around nokogiri are showing us that we have a need.

I don't think we should include Ruby 2.0.0 as that's no longer a supported version. I think a case could be made for dropping Ruby 1.9.3 too. Technically Ruby 2.1 is no longer supported, but I like to give people plenty of time to update.

mojavelinux commented:
I didn't realize I had left the build broken. If you rebase (and drop Ruby 1.9.3 and Ruby 2.0.0), we should be ready to merge.

miltador commented:
Yes, because of platform differences I think it would be cool to check for Windows builds.
I rebased and removed 1.9.3 and 2.0.0 from rubies list.
Should I open another PR to update Travis config dropping 1.9.3 and 2.0.0?

mojavelinux commented:
Thanks!

> Should I open another PR to update Travis config dropping 1.9.3 and 2.0.0?

We can't do that yet. I'm just not adding Ruby < 2.1 for anything new. I am waiting until 1.6.0 to officially drop old Rubies from the primary build.

miltador commented:
OK, got it. Thanks!
